### DISCUSSION TITLE
Is it possible to use Github pages on sub domain ?

### DISCUSSION THREAD

**Discussion by salimbenfarhat** [community/community#47521] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, all. It is possible to use Github pages with subdomain in addition to the domain?

**Comment by unknowVariable** [comment, marked as answer]
Hello
You can configure your subdomain as www or as a distinct section of your site, like blog.example.com . Subdomains are configured with a CNAME record through your DNS provider. For more information, see "Managing a custom domain for your GitHub Pages site." here : https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

**Comment by salimbenfarhat** [comment]
Hello and thank you @alixconstantin. Here is the response from my website hosting.

It is actually not allowed to add a CNAME to a domain/subdomain that already has an A and/or AAAA entry. This limitation is not decided by us but is part of the RFC concerning DNS zones: https://datatracker.ietf.org/doc/html/rfc1912#section-2.4

You would therefore have to choose between the CNAME or A/AAAA entry on your subdomain testsubdomain1

I set only the CNAME and the problem is solved.
